plugins/mac-virtualcam/src/dal-plugin/OBSDALPlugInInterface.mm

```diff
@@ -85,13 +85,15 @@ HRESULT HardwarePlugIn_QueryInterface(CMIOHardwarePlugInRef self, REFIID uuid,
 }
 
 // I think this is deprecated, seems that HardwarePlugIn_InitializeWithObjectID gets called instead
-OSStatus HardwarePlugIn_Initialize(CMIOHardwarePlugInRef self)
+OSStatus HardwarePlugIn_Initialize(CMIOHardwarePlugInRef self
+				   __attribute__((unused)))
```

Comment:
Since it's c++, could we just omit the name here (and the other instances)? We do that for browser a lot and it makes things more clean imo

Reply:
I don't think that will work in these specific cases, because the parameters are indeed used when the project is compiled in `Debug` config - otherwise the `DLogFunc` macro is empty and the parameters are not used.

If the macros themselves are restored you're back at square one because the macro itself then emits an "unused variable" warning for every invocation as its function block is empty. Might require an entirely different solution.